pom.xml

```diff
@@ -49,6 +49,7 @@
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
         <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
         <dependency.check.version>1.3.0</dependency.check.version>
+        <workflow-jenkins-plugin.version>1.4</workflow-jenkins-plugin.version>
```

Comment:
You can use [up to 1.7](https://github.com/jenkinsci/workflow-plugin/blob/master/CHANGES.md#17-may-29-2015) with this LTS.